File: 2271-rearrange-array-elements-by-sign/rearrange-array-elements-by-sign.cpp
class Solution {
public:
    vector<int> rearrangeArray(vector<int>& nums) {
        int i=0;
        int j=1;
        int n= nums.size();
        vector <int> ans (n);
        //Ashwini
        
        for (int k=0; k<n; k++){
            if (nums[k]>0){
                if (i<n){
                    ans[i]=nums[k];
                    i+=2;
                }
            }else{
                if (j<n){
                    ans[j]= nums[k];
                    j+=2;
                }
            }
        }

        return ans;
    }
};

File: 2271-rearrange-array-elements-by-sign/rearrange-array-elements-by-sign.py
class Solution(object):
    def rearrangeArray(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        positive_array= []
        negative_array= []
        ans=[]
        k=0
        n=len(nums)

        for i in range ((n)):
            if (nums[i]>0):
                positive_array.append(nums[i])

            else:
                negative_array.append(nums[i])


        while (k <n/2):
            ans.append(positive_array[k])
            ans.append(negative_array[k])
            k+=1

        return ans


        